src/find/azext_find/custom.py

```diff
@@ -25,23 +29,46 @@ def process_query(question):
         num_results_to_show = 1
         if answer_list:
```

Comment:
Is it possible to not have an answer list?

Reply:
I haven't see it happening, but I had the if, just in case.

Reply:
Show an error message in that case. Currently, that scenario will simply end the command with no explanation.

Reply:
done.